packages/nimble-components/src/select/tests/select-opened-matrix.stories.ts

```diff
@@ -17,16 +18,19 @@ export default metadata;
 
 const positionStates = [
     ['below', 'margin-bottom: 120px;'],
-    ['above', 'margin-top: 120px;']
+    ['above', 'margin-top: 180px;']
 ] as const;
 type PositionState = (typeof positionStates)[number];
 
+const filterModeStates = Object.values(FilterMode);
+type FilterModeState = (typeof filterModeStates)[number];
+
 // prettier-ignore
-const component = ([
-    position,
-    positionStyle
-]: PositionState): ViewTemplate => html`
-    <${selectTag} open position="${() => position}" style="${() => positionStyle}">
+const component = (
+    [position, positionStyle]: PositionState,
+    filterMode: FilterModeState
```

Comment:
Should there also be stories for the following?
- a filter string being set to make sure the `input` font/color is correct when something other than the placeholder is shown
- the "No items found" state

Reply:
I've added matrix tests that adds a filter, of which there is no match, and thus shows the 'No items found' text.